sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.0.0b2 (Unreleased)
+## 1.0.0b2 (2022-11-03)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Fixed bug in testing pipeline.
```

Comment:
This isn't a customer facing change (or even an internal one) and thus nothing should be added to the changelog.

Reply:
Okay. I just removed the entry then.